app/assets/javascripts/discourse/app/widgets/post-cooked.js

```diff
@@ -162,16 +161,14 @@ export default class PostCooked {
             !bestElements.has(onebox) ||
             bestElements.get(onebox) === link
           ) {
-            const title = I18n.t("topic_map.clicks", { count: lc.clicks });
-
-            link.appendChild(document.createTextNode(" "));
-            link.appendChild(
-              domFromString(
-                `<span class='badge badge-notification clicks' title='${title}'>${number(
```

Comment:
There's some CSS for `.clicks` and `.badge-notification.clicks` that should be removed/adapted to the new technique.